tests-perf/waf-rules/locust-trigger-rate-limit.py

```diff
@@ -0,0 +1,42 @@
+"""locust-trigger-rate-limit.py
```

Comment:
This looks great.  We should follow this pattern for the rest of the WAF rules if possible.  Its a nice way to document them AND have a way to ensure they work as designed.